policyengine_api/services/household_service.py

```diff
@@ -0,0 +1,128 @@
+import json
+from sqlalchemy.engine.row import LegacyRow
+
+from policyengine_api.data import database
+from policyengine_api.utils import hash_object
+from policyengine_api.constants import COUNTRY_PACKAGE_VERSIONS
+
+
+class HouseholdService:
+
+    def get_household(self, country_id: str, household_id: int) -> dict | None:
+        """
+        Get a household's input data with a given ID.
+
+        Args:
+            country_id (str): The country ID.
+            household_id (int): The household ID.
+        """
+        print("Getting household data")
+
+        try:
+            row: LegacyRow | None = database.query(
+                f"SELECT * FROM household WHERE id = ? AND country_id = ?",
```

Comment:
**followon/nonblocking** I think we already talked about this... but I think [ID is actually unique already](https://github.com/PolicyEngine/policyengine-api/blob/544b0d4f4e79b60ae930ccbd9a7ecb82c176ff9b/policyengine_api/data/README.md?plain=1#L22)  (isn't this an auto-increment) so we don't even need the country we're currently requiring.

Reply:
That is true